enos/ansible/roles/influx/tasks/main.yml

```diff
@@ -40,24 +46,20 @@
     delay: 2
     timeout: 120
 
-- name: Create the cadvisor database
-  shell: "curl -s -XPOST http://{{ influx_vip }}:8086/query --data-urlencode 'q=CREATE DATABASE cadvisor'"
-  register: result
-  until: result.stdout == '{"results":[{}]}'
-  retries: 10
-  delay: 2
+- name: Create the telegraf database
+  influxdb_database:
+    hostname: "localhost"
+    database_name: "telegraf"
```

Comment:
What happened to the cadvisor database ?

Reply:
oups :)